app/assets/stylesheets/cortex-plugins-core/components/thumbnail-placeholder.scss

```diff
@@ -0,0 +1,12 @@
+.thumbnail-placeholder {
+  background-color: #E4E4E4; // TODO: Abstract to cortex-style-base library. This should be $color-grey-evenlighter
```

Comment:
Even before the style base for Cortex is founded could you please extract this so we can just copy and paste the variables when the time comes